boards/arm/96b_ivy5661/pinmux.c

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Remove pinmux.c as this function isn't doing anything.

Reply:
If pinmux.c is removed, CMakeLists.txt is also unneeded, remove them all.